npm install error on Elementary OS (Ubuntu based)

Dea Team,

i am new aerospike when i installing module as instructed in ubuntu the following error is coming ,kindly suggest : 

gmchaturvedi@gmchaturvedi-HP-14-Notebook-PC:~/Documents/d/d1/Apps/client$ npm install aerospike

> aerospike@2.4.4 install /home/gmchaturvedi/Documents/d/d1/Apps/client/node_modules/aerospike
> node-gyp rebuild

make: Entering directory '/home/gmchaturvedi/Documents/d/d1/Apps/client/node_modules/aerospike/build'
  TOUCH binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate
  ACTION binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate
info: downloading 'https://artifacts.aerospike.com/aerospike-client-c/4.1.1/aerospike-client-c-libuv-devel-4.1.1-1.elementary0.4.x86_64.rpm' to '/home/gmchaturvedi/Documents/d/d1/Apps/client/node_modules/aerospike/aerospike-client-c/package/aerospike-client-c-libuv-devel-4.1.1-1.elementary0.4.x86_64.rpm'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 93220    0 93220    0     0   4717      0 --:--:--  0:00:19 --:--:-- 23665
scripts/aerospike-client-c.sh: line 70: [: ca69ba7be38bf3368ac5e64924de30a781d507de89a9481dbb31af21915d5c67: unary operator expected
error: invalid checksum for 'aerospike-client-c-libuv-devel-4.1.1-1.elementary0.4.x86_64.rpm'
aerospike-client-c.target.mk:16: recipe for target 'binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate' failed
make: *** [binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate] Error 1
rm binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate
make: Leaving directory '/home/gmchaturvedi/Documents/d/d1/Apps/client/node_modules/aerospike/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Linux 4.4.0-64-generic
gyp ERR! command "/usr/bin/nodejs" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/gmchaturvedi/Documents/d/d1/Apps/client/node_modules/aerospike
gyp ERR! node -v v6.10.0
gyp ERR! node-gyp -v v3.5.0
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.0.14 (node_modules/react-scripts/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.0.14: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/react-scripts/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

--- a/scripts/os_version
+++ b/scripts/os_version
@@ -112,7 +112,7 @@ main() {
 			distro_version=6
 		else
 			error "Unsupported linux distibution: $distro_id $distro_version"
-		exit 1
+			exit 1
 		fi
 		distro_long="centos${distro_version}"
 		distro_short="el${distro_version}"
@@ -122,11 +122,51 @@ main() {
 		distro_version=${distro_version%.*}
 		distro_short="${distro_id}${distro_version}"
 		;;
+	'elementary' )
+		case "$distro_version" in
+		"0.4" )
+			distro_version="16.04"
+			;;
+		* )
+			error "Unsupported linux distibution version: $distro_id $distro_version"
+			error "Guessing compatibility with Ubuntu 16.04"
+			distro_version="16.04"
+			;;
+		esac
+		distro_id="ubuntu"
+		distro_long="${distro_id}${distro_version}"
+		distro_version=${distro_version%.*}
+		distro_short="${distro_id}${distro_version}"
+		;;
+	'linuxmint' )
+		case "$distro_version" in
+		"17"* )
+			distro_version="14.04"
+			;;
+		"18"* )
+			distro_version="16.04"
+			;;
+		* )
+			error "Unsupported linux distibution version: $distro_id $distro_version"
+			error "Guessing compatibility with Ubuntu 16.04"
+			distro_version="16.04"
+			;;
+		esac
+		distro_id="ubuntu"
+		distro_long="${distro_id}${distro_version}"
+		distro_version=${distro_version%.*}
+		distro_short="${distro_id}${distro_version}"
+		;;
 	'amzn' )
 		distro_long="ami"
 		distro_short="ami"
 		;;
 	* )
+		if [ "$ID_LIKE" ]
+		then
+			distro_id=$(echo "$ID_LIKE" | cut -d" " -f1)
+			distro_version="_unknown"
+		fi
 		distro_long="${distro_id}${distro_version}"
 		distro_short="${distro_id}${distro_version}"
 		;;

